### PR TITLE
mogrify: add page

### PR DESCRIPTION
**Pull request by sakibjunaed**
Fixed issue #9531

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/mogrify.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Set background color:
+
+`mogrify -background color {{input-file}}`
+
+- Set image format type:
+
+`mogrify -format type {{input-file}}`
+
+- Surround image with border:
```
Comment (kbdharun) [inline pages/linux/mogrify.md:18]:
```suggestion
- Surround the image with a border:
```

File: pages/linux/mogrify.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`mogrify -border geometry {{input-file}}`
+
+- Flip image vertically:
```
Comment (kbdharun) [inline pages/linux/mogrify.md:22]:
```suggestion
- Flip the image vertically:
```

File: pages/linux/mogrify.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# mogrify
+
+> Edit and transform an image by overwriting the original file.
+> More information: <https://manned.org/mogrify>.
+
+- Join images into a single multi-image file:
+
+`mogrify -adjoin {{input-file}} {{input-file}}`
```
Comment (kbdharun) [inline pages/linux/mogrify.md:8]:
```suggestion
`mogrify -adjoin {{input_file}} {{input_file}}`
```

File: pages/linux/mogrify.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Set background color:
+
+`mogrify -background color {{input-file}}`
```
Comment (kbdharun) [inline pages/linux/mogrify.md:12]:
```suggestion
`mogrify -background color {{input_file}}`
```

File: pages/linux/mogrify.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Set image format type:
+
+`mogrify -format type {{input-file}}`
```
Comment (kbdharun) [inline pages/linux/mogrify.md:16]:
```suggestion
`mogrify -format type {{input_file}}`
```

File: pages/linux/mogrify.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Surround image with border:
+
+`mogrify -border geometry {{input-file}}`
```
Comment (kbdharun) [inline pages/linux/mogrify.md:20]:
```suggestion
`mogrify -border geometry {{input_file}}`
```

File: pages/linux/mogrify.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Join images into a single multi-image file:
+
+`mogrify -adjoin {{input-file}} {{input-file}}`
+
+- Set background color:
+
+`mogrify -background color {{input-file}}`
+
+- Set image format type:
```
Comment (kbdharun) [inline pages/linux/mogrify.md:14]:
```suggestion
- Set the image format type:
```

File: pages/linux/mogrify.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# mogrify
+
+> Edit and transform an image by overwriting the original file.
+> More information: <https://manned.org/mogrify>.
+
+- Join images into a single multi-image file:
+
+`mogrify -adjoin {{input-file}} {{input-file}}`
+
+- Set background color:
```
Comment (kbdharun) [inline pages/linux/mogrify.md:10]:
```suggestion
- Set the background color:
```

File: pages/linux/mogrify.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`mogrify -format type {{input-file}}`
+
+- Surround image with border:
+
+`mogrify -border geometry {{input-file}}`
+
+- Flip image vertically:
+
+`mogrify -flip {{input-file}}`
```
Comment (kbdharun) [inline pages/linux/mogrify.md:24]:
```suggestion
`mogrify -flip {{input_file}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/mogrify.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+# mogrify
+
+> Edit and transform an image by overwriting the original file.
+> More information: <https://manned.org/mogrify>.
+
+- Join images into a single multi-image file:
+
+`mogrify -adjoin {{input_file}} {{input_file}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/mogrify.md:8]:
```suggestion
`mogrify -adjoin {{path/to/file1 path/to/file2 ...}}`
```

Mandatory fix: [path format is incorrect](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).
Mandatory fix: command syntax is incorrect: not just 2 arguments expected.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
At least one mandatory fix detected, changes requested.
Page should be scanned for multiple violations of similar issues (suggestions are created just for their first occurrences).
Thanks for a contribution! 🚀

**Comment by Waples** [comment]
@sakibjunaed are you still working on this or shall somebody else apply the changes?

**Pull request closed without merging**